diktat-rules/src/test/resources/test/paragraph3/file_structure/OtherCommentsTest.kt

```diff
@@ -0,0 +1,11 @@
+// some notes on this file
```

Comment:
Why `COMMENT_BLOCK` doesn't move exactly like the comment

Reply:
If there is a `BLOCK_COMMENT` before code, it's considered as containing copyright information. But you are right, I need to check copyright keywords too, like we do in some other rule.